.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+name: ci
+
+on:
+  pull_request:
+  push:
+    branches:
+    - master
+
+permissions:
+  contents: read
+
+jobs:
+  lint:
+    runs-on: ubuntu-22.04
+    steps:
+    - name: ci/checkout-repo
+      uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0
+      with:
+        fetch-depth: 0
+
```

Comment:
Would suggest removing these NL. But feel free to ignore, since this is personal preference.

Reply:
My personal preference is having them, actually 😅